uint8 target_id
float64 confidence
geometry_msgs/PoseStamped pose